crates/oxc_linter/src/config/config_builder.rs

```diff
@@ -147,15 +147,18 @@ impl ConfigStoreBuilder {
         let (oxlintrc, extended_paths) = resolve_oxlintrc_config(oxlintrc)?;
 
         // Collect external plugins from both base config and overrides
-        let mut external_plugins = FxHashSet::default();
+        let mut external_plugins: FxHashSet<(Option<&Path>, &str)> = FxHashSet::default();
```

Comment:
This is neat! Sort of what I had in mind, but I was trying to make it work with a rust enum. That was getting me nowhere.